contracts/Colony.sol

```diff
@@ -61,6 +61,23 @@ contract Colony is ColonyStorage {
     });
   }
 
+  function bootstrapColony(uint _amountPerUser, address[] _users) public
```

Comment:
I suspect that people will very commonly want to bootstrap their colony such that users have different amounts of starting reputation from each other. Why not take an array of `uint`s as well? In that scenario, don't worry about figuring out at the start whether there are enough tokens - get `balanceOf(this)` at the start, and track what it is as you sent tokens out; if not enough would remain for the next transfer, then `mint` some.

Alternatively, it would be more gas efficient to just ignore minting in this function at all, and we expect them to call `mint` before calling this function if they see that it's going to be necessary. I am aware this would be contrary to what I told you originally via Gitter - apologies for that. I hadn't properly thought about whether one could efficiently implement what I was asking for. I think I would prefer that implementation, overall.

Reply:
Thanks, i agree that tokens should be minted outside of this function.